admin-api/central_ledger_api_v1.0.json

```diff
@@ -0,0 +1,3355 @@
+            },
+            "RecordFundsOut": {
+                "type": "object",
+                "properties": {
+                    "action": {
+                        "type": "string",
+                        "enum": [
+                            "recordFundsOutCommit",
+                            "recordFundsOutAbort"
+                        ]
+                    },
+                    "reason": {
+                        "type": "string"
+                    }
+                },
+                "required": [
+                    "action",
+                    "reason"
+                ]
+            },
+            "Refund": {
+                "title": "Refund",
+                "type": "object",
+                "description": "Data model for the complex type Refund.",
+                "properties": {
+                    "originalTransactionId": {
+                        "type": "string",
+                        "pattern": "^[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}$",
+                        "description": "Reference to the original transaction id that is requested to be refunded."
+                    },
+                    "refundReason": {
+                        "type": "string",
+                        "minLength": 1,
+                        "maxLength": 128,
+                        "description": "Free text indicating the reason for the refund."
+                    }
+                },
+                "required": [
+                    "originalTransactionId"
+                ]
+            },
+            "Service": {
+                "title": "Service",
+                "type": "object",
+                "description": "Data model for Service.",
+                "properties": {
+                    "name": {
+                        "type": "string",
+                        "description": "Service name.",
+                        "example": "datastore"
+                    },
+                    "status": {
+                        "type": "string",
+                        "enum": [
+                            "OK",
+                            "DOWN"
+                        ],
+                        "description": "Below are the allowed values for the enumeration. - OK - The service is up and running. - DOWN: The service is down.",
+                        "example": "OK"
+                            }
+                },
+                "required": [
+                    "name",
+                    "status"
+                ]
+            },
+            "Services": {
+                "title": "Services",
+                "description": "Data model for Services.",
+                "type": "array",
+                "items": {
+                    "$ref": "#/components/schemas/Service"
+                }
+            },
+            "SettlementModel": {
+                "type": "object",
+                "required": [
+                    "autoPositionReset",

```

Comment:
Please add `currency` to the `required` list.

This is a change being made by the CGS changes introduced by @vgenev and @shashi165.

@elnyry-sam-k FYI.